include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -194,15 +194,18 @@ class pairwise_alignment_edit_distance_unbanded
 public:
 
     /*!\name Constructors, destructor and assignment
-     * The copy-constructor, move-constructor, copy-assignment, move-assignment,
-     * and destructor are implicitly defined.
      * \{
      */
+     //!\brief Deleted due to type deduction 
```

Comment:
Isn't it deleted because you need to provide the database and such?
There is no type deduction here